Jenkinsfile

```diff
@@ -139,48 +151,25 @@ podTemplate(name: 'fedora-atomic-inline', label: 'fedora-atomic-inline', cloud:
                         messageContent = ''
                         sendMessage(messageProperties, messageContent)
 
-                        // Provision of resources
-                        allocDuffy("${current_stage}")
-
-                        echo "Duffy Allocate ran for stage ${current_stage} with option ${env.DUFFY_OP}\r\n" +
-                                "ORIGIN_WORKSPACE=${env.ORIGIN_WORKSPACE}\r\n" +
-                                "ORIGIN_BUILD_TAG=${env.ORIGIN_BUILD_TAG}\r\n" +
-                                "ORIGIN_CLASS=${env.ORIGIN_CLASS}"
-
                         job_props = "${env.ORIGIN_WORKSPACE}/job.props"
                         job_props_groovy = "${env.ORIGIN_WORKSPACE}/job.groovy"
                         convertProps(job_props, job_props_groovy)
                         load(job_props_groovy)
 
-                        // Stage resources - RPM build system
-                        setupStage("${current_stage}")
-
-                        if (env.OSTREE_BRANCH == null) {
-                            env.OSTREE_BRANCH = ""
+                        // Call rpmbuild-test.sh
+                        // in rpmbuild container
+                        container('rpmbuild') {
+                            sh "fed_repo=${env.fed_repo} " +
+                               "fed_branch=${env.fed_branch} " +
+                               "fed_rev=${env.fed_rev} " +
+                               "/home/rpmbuild-test.sh"
```

Comment:
We will also need RSYNC_PASSWORD, enable_rsync=yes and production=yes (my apologies if they are in here somewhere else and I missed them)